packages/cli/src/controllers/project.controller.ts

```diff
@@ -210,44 +216,108 @@ export class ProjectController {
 	@Patch('/:projectId')
 	@ProjectScope('project:update')
 	async updateProject(
-		req: AuthenticatedRequest,
+		_req: AuthenticatedRequest,
 		_res: Response,
 		@Body payload: UpdateProjectDto,
 		@Param('projectId') projectId: string,
 	) {
-		const { name, icon, relations, description } = payload;
+		const { name, icon, description } = payload;
+		// Only update scalar settings on this endpoint. Member updates are handled by dedicated endpoints.
 		if (name || icon || description) {
 			await this.projectsService.updateProject(projectId, { name, icon, description });
 		}
-		if (relations) {
-			try {
-				const { project, newRelations } = await this.projectsService.syncProjectRelations(
-					projectId,
-					relations,
-				);
+	}
```

Comment:
Why are we removing the relations field in this case? As far as I can tell the API would still accept the relations field, but we would ignore it at this point?